CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.39.120](https://github.com/brave/brave-browser/releases/tag/v1.39.120)
+
+ - Changed position of VPN toggle under settings menu. ([#23240](https://github.com/brave/brave-browser/issues/23240))
+ - Fixed Brave Firewall + VPN crash when device attempts connecting to Google Play Store after losing network connectivity in certain cases. ([#23177](https://github.com/brave/brave-browser/issues/23177))
+ - Fixed subscription status under Brave Firewall + VPN settings not specifying if it's a trial, monthly or yearly subscription. ([#23004](https://github.com/brave/brave-browser/issues/23004))
+ - Fixed GenerateCryptoErrorForTypes error which caused Brave Sync to fail in certain cases. ([#22898](https://github.com/brave/brave-browser/issues/22898))
```

Comment:
How about `Fixed errors for sync decryption which caused Brave Sync to fail in certain cases`? rather than using the error type message?

Reply:
Done 👍